spec/customer_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'customer'
+
+describe Customer do
+
+  context '#menu' do
+
+    it 'can see the menu' do
+      allow(subject).to receive(:menu).and_return(:Burger)
+      expect(subject.menu).to eq :Burger
+    end
+  end
+
+  context '#place_order' do
+
+    before { subject.place_order :Burger }
+
+    it 'will raise an error when dish is not on the menu' do
+      expect { subject.place_order(:Duck) }.to raise_error 'Please select a dish from the menu.'
+    end
+
+    it 'can place an order' do
+      expect(subject.place_order :Burger).to eq 'Thank you for ordering.'
+    end
+
+    it 'a dish has a default quantity of one' do
+      expect(subject.order.values).to eq [1]
+    end
+
+    it 'a dish can be ordered more than once at the same time' do
+      subject.place_order :Burger, 3
+      expect(subject.order.values).to eq [4]
+    end
+
+    it 'can take multiple orders for the same dish' do
+      subject.place_order :Burger, 2
+      expect(subject.order.values).to eq [3]
+    end
+
+    it 'can order different dishes' do
+      subject.place_order :Chips, 4
+      subject.place_order :Salad
+      expect(subject.order.values).to eq [1, 4, 1]
+    end
+  end
+
+  context '#review_order' do
+
+    it 'raises an eror if no dishes are ordered' do
+      expect { subject.review_order }.to raise_error "You haven't ordered anything."
```

Comment:
Metrics/LineLength: Line is too long. [84/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)